types/time.go

```diff
@@ -1664,7 +1670,10 @@ func parseSingleTimeValue(unit string, format string, strictCheck bool) (int64,
 		}
 		dayCount := iv / int64(GoDurationDay/gotime.Second)
 		iv %= int64(GoDurationDay / gotime.Second)
-		return 0, 0, dayCount, iv*int64(gotime.Second) + dv*int64(gotime.Microsecond), err
+		if sign == int64(-1) {
+			return 0, 0, dayCount, -(iv*int64(gotime.Second) + dv*int64(gotime.Microsecond)), err
+		}
+		return 0, 0, dayCount, (iv*int64(gotime.Second) + dv*int64(gotime.Microsecond)), err
```

Comment:
Please remove the redundant brackets.

Reply:
If you prefer to just fix the `SECOND` unit problem, please change your pull request title more precisely and do not use `neg` for abbreviation.